ios/HathorMobile/HathorMobile.entitlements

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>aps-environment</key>
+	<string>development</string>
```

Comment:
This one, we should change when releasing.

Reply:
Right! I can change it, no problem. But to test on mainnet I will need the GoogleService-Info.plist resource for mainnet as well.

Reply:
Right! Ok, I can change it now.

Reply:
Changed to production at https://github.com/HathorNetwork/hathor-wallet-mobile/pull/268/commits/67ee4e63d11736cd78932e55234c1a988841a66d